packages/server/lib/controllers/sync.controller.ts

```diff
@@ -105,7 +105,7 @@ class SyncController {
 
     public async trigger(req: Request, res: Response<any, Required<RequestLocals>>, next: NextFunction) {
```

Comment:
in an ideal world I would have refactored all the endpoints to use the `Endpoint` type and zod validation, unfortunately because of a lack of time I've decided not to do it now

Reply:
wish I had done them sooner